OpenRA.Mods.Common/Traits/Transforms.cs

```diff
@@ -97,6 +98,40 @@ public bool CanDeploy()
 			return buildingInfo == null || self.World.CanPlaceBuilding(self.Location + Info.Offset, actorInfo, buildingInfo, self);
 		}
 
+		IEnumerable<Order> ClearBlockersOrders(World world, CPos topLeft)
+		{
+			var allTiles = buildingInfo.Tiles(topLeft).ToArray();
```

Comment:
```suggestion
			var allTiles = buildingInfo.Tiles(topLeft).ToList();
```

Not that it matters but a ToList is faster here